license.md

```diff
@@ -1,5 +1,6 @@
 MIT License
 
+Copyright (c) 2020 Vercel, Inc.
 Copyright (c) 2017 Zeit, Inc.
```

Comment:
```suggestion
Copyright (c) 2021 Vercel, Inc.
```

Reply:
AFAIK when editing a licnese file the old copyright MUST be kept there. You just need to add the new copyright holder

Reply:
It's the same holder - Vercel used to be ZEIT. You can see this change reflected on other repos in our org.